### PR TITLE
ENG-501: Allow array of public keys

### PR DESCRIPTION
**Pull request by timmyjose**
This is the PR 1 for the feature, ENG-501 - allow array of public keys.

Changes made:
  * Updated lalrpop grammar.
  * Added test.
  * Bumped version.
  * (Minor) updated pull-collections.sh script to account for macOS differences.

### CODE REVIEW

File: js/lerna.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
   "packages": [
     "./"
   ],
-  "version": "0.4.10"
```
Comment (calummoore) [inline js/lerna.json:8]: We have automate CI for bumping, so no need to bump manually. Can you revert this.

File: js/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@polybase/polylang",
-  "version": "0.4.10",
```
Comment (calummoore) [inline js/package.json:3]: We have automate CI for bumping, so no need to bump manually. Can you revert this.

**Review by calummoore — COMMENTED** [review]
Looks good, please just revert the version numbers :)

**Comment by calummoore** [comment]
Fyi, once these updates are in, and I merge this change it will create a `beta` release on npm, which you can then update your dependencies with to ensure they are passing CI.

File: js/lerna.json (hunk 1 above)
Comment (timmyjose) [reply js/lerna.json:8]: Thank you! I had a suspicion that that was the case. I'll revert the version numbers in a push.

**Comment by timmyjose** [comment]
> Fyi, once these updates are in, and I merge this change it will create a `beta` release on npm, which you can then update your dependencies with to ensure they are passing CI.

Okay, that makes sense. I'll track it and update the dependencies in the other PRs (and explorer as well, in a separate new PR).

File: js/package.json (hunk 2 above)
Comment (timmyjose) [reply js/package.json]: Done!

File: js/lerna.json (hunk 1 above)
Comment (timmyjose) [reply js/lerna.json]: Done.

**Review by calummoore — APPROVED** [review]

**Pull request merged**